docs/source/reference/kubernetes/kubernetes-troubleshooting.rst

```diff
@@ -68,7 +68,19 @@ Run :code:`sky check` to verify that SkyPilot can access your cluster.
 If you see an error, ensure that your kubeconfig file at :code:`~/.kube/config` is correctly set up.
 
 
-Step A3 - Can you launch a SkyPilot task?
+Step A3 - Do your nodes have enough disk space?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your nodes are out of disk space, pulling the SkyPilot images may fail with :code:`rpc error: code = Canceled desc = failed to pull and unpack image: context canceled`.
+Make sure your nodes are not under disk pressure by checking :code:`Conditions` in :code:`kubectl describe nodes`, or by running:
```

Comment:
Could we elaborate a bit for where the error message will show? Is it in the `provision.log` or the terminal?

Reply:
updated!